src/core/psi_container.jl

```diff
@@ -250,10 +250,16 @@ function decode_symbol(name::Symbol)
     return split(String(name), PSI_NAME_DELIMITER)
 end
 
+# TODO: remove once all references are changed
 constraint_name(cons_type, device_type) = encode_symbol(device_type, cons_type)
 constraint_name(cons_type) = encode_symbol(cons_type)
+make_constraint_name(cons_type, device_type) = encode_symbol(device_type, cons_type)
+make_constraint_name(cons_type) = encode_symbol(cons_type)
+# TODO: remove once all references are changed
 variable_name(var_type, device_type) = encode_symbol(device_type, var_type)
 variable_name(var_type) = encode_symbol(var_type)
+make_variable_name(var_type, device_type) = encode_symbol(device_type, var_type)
```

Comment:
I could make global changes in a separate PR.  For now I made changes in files that I was refactoring as part of this PR.

Reply:
We can include these changes in the same PR since they are a general refactor of the variable creation/handling

Reply:
OK.  I'll wait until we reach agreement on the changes and do this at the end.

Reply:
Let's do this in another PR